### DISCUSSION TITLE
Call for testing and translation for Freeplane 1.12.2 stable

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#1966] [Announcements]
Dear community, please test the latest ≈. If you do not report any bugs, I am going to release it as stable soon.

**Comment by aaa1386** [comment]
The `Unfold node on paste` option, whether active or inactive, does not have a precise function when the `synchronize selection` option is active and the user has opened two views of the map.
See also this idea:
#1968

**Reply by dpolivaev** [reply]
I do not understand how you mean it. Please open a new bug report and add as many screen shots and mind map files as needed to completely explain which behavior you observe and why you find it wrong

**Reply by aaa1386** [reply]
We have activated the "Unfold node on paste" option
When we drag and drop a node from the right view to a folded node in the left view, that node does not unfold.
 This bug is due to the fact that the views are activated only by clicking and cannot be clicked in drag and drop mode. Maybe idea #1968 can solve this problem. In addition, it is a very easy idea.

In some other cases, I thought that FP was working incorrectly, but I was wrong because of the two modes of delay and no delay in dropping the node.

**Reply by aaa1386** [reply]
Now I have completed the idea of ​​#1968

**Comment by aaa1386** [comment]
When two views of the map are open and the user has activated the "Scroll map after selecting a node" option. If the user selects a node from the right view, a node will be selected by FP in the left view. Isn't it better for FP to scroll the left view by the "Scroll map after selecting a node" option?

thank

**Comment by aaa1386** [comment]
https://github.com/user-attachments/assets/6d3bc3e0-8929-492e-9d46-b3845787dbfa

**Reply by aaa1386** [reply]
The left view is filtered with the "important" style. When we paste node 1 into node 3, the left view shows node 1 while not having the "important" style.

**Reply by dpolivaev** [reply]
1. This specific issue does not look like a bug because filters are automatically applied only to not selected views, and when you pasted a node to the view on the left it was  or became selected.
2. The node 3 on the left view has already been unfolded, and therefore setting "unfold on paste" does not apply either.
3. If you want to continue discussion about this particular case please file a NEW bug report issue because this discussion thread is not thought for discussing new bugs. Its only purpose is to ask people to report the bugs they find in the preview versions, and it is usually done by reporting new issues.

**Comment by Mushy-Snugglebites-badonkadonk** [comment]
@dpolivaev Hi Dimitry, I wanted to connect with you and your team to try to understand why the Apple DMGs are [not signed](https://www.virustotal.com/gui/file/be54bd7719bb82dbbd652eaeb1cda2eb459629602dd2b828dea7630fa0f9f378/details). As a result, Apple throws a warning when launching the application. Would love to move back to Freeplane for future proofing instead of Xmind. I imagine the community would feel safer if Freeplane releases were signed with an Apple developer key.

**Reply by dpolivaev** [reply]
Signed. Even notarized successfully. If you see anything different, submit a bug report and upload your screenshots.
